channeldb/kvdb/kvdb_test_bdb.go

```diff
@@ -0,0 +1,5 @@
+// +build !kvdb_etcd
```

Comment:
👍 

Once we want them to run on travis (both versions), we'll also need to update the script we use to execute the buid.